src/components/Wizard/SpotlightWizard/SpotlightActionInput.tsx

```diff
@@ -44,6 +44,17 @@ export default function SpotlightActionInput({
           disabled={disabled}
         />
       </FormRow>
+      <FormRow
+        label="Dismiss?"
+        containerClassName="form-input-check"
+        helpText="Dismiss the spotlight when this action triggers"
+      >
+        <RegisteredFormCheck
+          name={`${controlPrefix}.dismiss`}
+          register={register}
+          disabled={disabled}
+        />
+      </FormRow>
```

Comment:
Navigation and dismissal should be *mutually exclusive*. You can get the current values of the other field using `watch()`.

Reply:
So I've done this with `disabled` and it works, but I don't like it. :P I may refactor it to be radio buttons.